include/numerics/petsc_solver_exception.h

```diff
@@ -89,6 +89,10 @@ class PetscSolverException : public SolverException
     LIBMESH_CHKERRQ(ierr);                      \
   } while (0)
 
+// Remove me: for backward compatibility with MOOSE only
+#define LIBMESH_CHKERR(ierr) LIBMESH_CHKERRQ(ierr)
+#define LIBMESH_CHKERR2(comm, ierr) LIBMESH_CHKERRA(comm, ierr)
```

Comment:
It's confusing that the one vs. two argument macros are equivalent to `PetscCall` and `PetscCallAbort` respectively. Is this only because MOOSE is "being dumb"?

Reply:
Sorry, I don't think I understood the question, `PetscCall` and `PetscCallAbort` are also one and two argument macros?

Reply:
`PetscCallAbort` will abort immediately if the error code is non-zero. This does not happen with `PetscCall`

Reply:
Agreed. Point being? (sorry, it's late over here...)

Reply:
I would not expect that `LIBMESH_CHKERR` and `LIBMESH_CHKERR2` have fundamentally different behavior when the only difference is that one takes a comm and the other does not

Reply:
Right, thanks, apologies. I'm with you now.

Yes, it's true that before, *if* `LIBMESH_ENABLE_EXCEPTIONS` wasn't defined, they'd both wind up calling `CHKERRABORT`, and that behaviour is now reserved to `LIBMESH_CHKERRA`. Note, however, that the fallback is only defined for the case where `LIBMESH_ENABLE_EXCEPTIONS` is indeed defined, for which the behaviour was already the same before.

Reply:
Got ya. Yea I thought you didn't likely *add* any silliness that wasn't there before. Still would be nice if we as a collective make this more consistent. Probably can do this after updating MOOSE